app/src/main/java/com/example/pokedexdesafio/data/model/PokemonDetailResponse.kt

```diff
@@ -16,21 +17,108 @@ data class PokemonDetail(
 
 //Type
 @Parcelize
-data class TypeContainer(val slot: Int, val type: Type) : Parcelable
+data class TypeContainer(val slot: Int?, val type: Type?) : Parcelable
```

Comment:
Rename this classes to end with Response

Reply:
solved